src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -157,7 +157,7 @@ private enum ResponseStatus
 
             if (authenticationRequestParameters.AppConfig.MultiCloudSupportEnabled)
             {
-                authParams.Properties["discover"] = "home";
+                authParams.Properties["instance_aware"] = "true";
```

Comment:
Do we still have ["discover"] = "home", if so what is that used for?

Reply:
"discover" = "home" is the magic sauce if you talk directly to WAM. 
"instance_aware" = "true" is the magic sauce if you talk to MSAL C++ or directly to ESTS. MSAL C++ will transform "instace_aware"="true" into "discover"="home" when talking to WAM on its own.